app/presenters/nsm/check_answers/read_only_report.rb

```diff
@@ -125,7 +125,7 @@ def further_information_section
       private
 
       def case_disposal_form
-        ycf_flow = FeatureFlags.youth_court_fee.enabled? && claim.rep_order_date >= Constants::YOUTH_COURT_CUTOFF_DATE
+        ycf_flow = FeatureFlags.youth_court_fee.enabled? && claim.nsm? && claim.!before_youth_court_cutoff?
```

Comment:
`foo.!bar` isn't a thing in Ruby, I'm afraid. You need `!foo.bar`

Reply:
God knows what I was thinking